payjoin/src/receive/optional_parameters.rs

```diff
@@ -75,7 +75,7 @@ impl Params {
                                 None
                             }
                         },
-                ("minfeerate", fee_rate) =>
+                ("minfeerate", fee_rate) => {
```

Comment:
These bracket changes seem like a lint. Were these fixes autogen or is this your preferred syntax or something else?

Reply:
definitely autogen, I have been struggling a bit after I reset my dev environment to try out nixos